packages/view-util/package.json

```diff
@@ -4,11 +4,13 @@
   "type": "module",
   "main": "index.js",
   "dependencies": {
-    "@rdfjs/dataset": "^2.0.0",
+    "@rdfjs/term-map": "^2",
+    "@rdfjs/traverser": "^0.1.0",
```

Comment:
I think traverser is already included in rdf-ext 2.1.0

 https://github.com/rdf-ext/rdf-ext/blob/4416a090d3f2df36fd7a2a1c9e588e6b25b4a974/index.js#L41

Reply:
Thanks. I was not using rdf-ext to begin with and this remained here